tools/ocamlmklib.ml

```diff
@@ -25,10 +25,8 @@ let mklib out files opts =
     then "-machine:AMD64 "
     else ""
   in
-  Printf.sprintf "link -lib -nologo %s-out:%s %s %s"
-                 machine out opts files
-  else Printf.sprintf "%s rcs %s %s %s && %s %s"
-                      Config.ar out opts files Config.ranlib out
+  Printf.sprintf "link -lib -nologo %s-out:%s %s %s" machine out opts files
+  else Printf.sprintf "%s rcs %s %s %s" Config.ar out opts files
```

Comment:
The `s` flag might be unnecessary and not available everywhere, according to https://pubs.opengroup.org/onlinepubs/9699919799/utilities/ar.html .  Removing the `s` flag would be consistent with the other invocations of `ar`.